engine/src/w32spec.cpp

```diff
@@ -98,10 +98,59 @@ void MCS_loadfile(MCExecPoint &ep, Boolean binary)
 #ifdef /* MCS_savefile_dsk_w32 */ LEGACY_SYSTEM
 void MCS_savefile(const MCString &fname, MCExecPoint &data, Boolean binary)
 {
-	if (!MCSecureModeCanAccessDisk())
+	char *t_path = NULL;
```

Comment:
Yes, it removed the `if (!MCSecureModeCanAccessDisk)...` statement - but since `w32spec.cpp` is no longer compiled on 7.0, I just took -theirs when merging. I can reinstate the `if` statement if needed though.